build.gradle

```diff
@@ -27,7 +30,12 @@ sourceSets {
   }
   test {
     java {
-      srcDirs = ['test']
+      srcDirs = ['test/src']
+    }
+  }
+  testFixtures {
+    java {
+      srcDirs = ['test/shared']
     }
     resources {
       srcDirs = ['test/testResources']
```

Comment:
A little out of scope and very nitpicky, but can this `test/resources` instead?

Reply:
This makes sense to me.  @jakobwenzel do you mind if we refactor this?

Reply:
Go ahead, I didn't think to rename it after moving it from the root directory.

Reply:
Just refactored it, thanks.